src/Bookings.js

```diff
@@ -1,20 +1,39 @@
-import React from "react";
-import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import React, { useState, useEffect } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then(res => res.json())
+      .then(data => {
+        console.log(data);
```

Comment:
Generally you want to avoid using `console.log` in project where it's not just you working on it, because it tends to clutter up the console output.

I'll show you how to use the debugger in Chrome developer tools so you can get the same information without using `console.log`